configs/default/FOXE-FOXEERF405V2.config

```diff
@@ -116,6 +120,9 @@ feature RX_SERIAL
 feature TELEMETRY
 feature OSD
 
+# aux
+aux 2 40 2 1600 2100 1
```

Comment:
Suggest adding `set box_user_1_name` to clarify what this setting is used for.

Reply:
Can you commit it @haslinghuis  I’m not with my computer all day

Reply:
What function is it used for? VTX_POWER ?

Reply:
Pino switch

Reply:
For what function? So we can label it

Reply:
Yeah I guess VTX power
Never use it 😂

Reply:
set box_user_1_name = CAM_SWITCH, it is cam switcher, we integrate the 2 way video switcher in the FC